### ISSUE TITLE
Maps not initializing / rendering

### ISSUE THREAD

**Issue by dgorissen** [yurukov/dc.leaflet.js#1]
Thanks for releasing this. Im using this with the latest dc version (2.0 alpha 2) and maybe thats the problem but I am unable to get the marker chart to actually render/show. dc.renderAll() triggers all other charts execept the leaflet ones. 

Invoking render() / draw() directly on the chart object has no effect either. Added various breakpoints to the code but those are never hit.

**Comment by pbadenski** [comment]
I had the same problem and @monostop fix works perfect. +1 for merging it into main dc.leaflet.js repository

**Comment by gordonwoodhull** [comment]
Yes, there was a fair amount of interface drift in dc.js between 1.7 and 2.0.

https://github.com/dc-js/dc.leaflet.js is compatible with dc.js 2.0
